fanficfare/adapters/adapter_inkbunnynet.py

```diff
@@ -203,7 +203,7 @@ def getChapterText(self, url):
 
     def before_get_urls_from_page(self,url,normalize):
         # To display the links to stories that are not available to guests.
-        if self.getConfig("username") and self.getConfig("always_login"):
```

Comment:
I'll let it slide, since it's a one line change, but please don't put unrelated changes in the same PR in future.